guardian/manage-frontend
# Product switch price formatting

## What does this change?
On Switch Options, Switch Complete and Switch Review, where it shows a whole monetary amounts in headers and text any trailing 0s are removed, and decimals are rounded to two places. For example £5.90 and £5.



## How to test
Any monetary value should appear in the Switch Option, Switch Complete  and Switch Review Storybook test. Since this remains behind a feature switch it can't be tested in production but on CODE (if the feature flag has been lifted) follow the product switch journey and you will see the value is fixed to two decimal places on the switch review page and the switch complete page.


## Images
**Switch Option** 
<img width="894" alt="image" src="https://user-images.githubusercontent.com/115992455/218511565-278ede32-4760-494a-b383-60fcc7ed5d1c.png">

<img width="884" alt="image" src="https://user-images.githubusercontent.com/115992455/218511623-a5f85abb-9989-437d-97fd-6b0b9784906b.png">

**Switch Review**
<img width="958" alt="image" src="https://user-images.githubusercontent.com/115992455/218511746-d7f2a0eb-8ae4-459e-8efb-0c5ad6690476.png">

**Switch Complete**
<img width="886" alt="image" src="https://user-images.githubusercontent.com/115992455/218511468-3397090d-bf14-4706-ac15-b3e27303d7a0.png">




## Accessibility
